tests/v2/validation/snapshot/snapshot_restore_test.go

```diff
@@ -62,6 +63,10 @@ func (s *SnapshotRestoreTestSuite) TestSnapshotRestoreETCDOnly() {
 }
 
 func (s *SnapshotRestoreTestSuite) TestSnapshotRestoreETCDOnlyDynamicInput() {
+	if s.clustersConfig.SnapshotRestore != "none" {
+		s.T().Skip()
```

Comment:
What if the configuration isn't the config file at all?

Reply:
Resolved